apps/bfd-data-npi/src/test/java/gov/cms/bfd/data/npi/utility/AppTest.java

```diff
@@ -0,0 +1,63 @@
+package gov.cms.bfd.data.npi.utility;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Please import only what is needed here and avoid star imports

Reply:
done

Reply:
done